### DISCUSSION TITLE
[ARCHIVED] Augmented Exercise 2 - Getting started with ARFoundation

### DISCUSSION THREAD

**Discussion by periode** [periode/augmenting-gallery#9] [Homework]
Turn your first assignment into an AR App, by taking into account the feedback you've received for the first assignment.

Full guidelines [here](https://github.com/periode/augmenting-gallery/wiki/Augmented-Exercises#exercise-2---getting-started-with-arkit).

Post:
- A video of your app working
- A link to the Unity project as a zip file
- A short paragraph about your intent and reflections.

**Comment by mdl601** [comment]
Michelle Leidecker

Logistically, this feels like it should have been easier than it was. I explored the use of the cell phone in society, which was ironic in the sense of trying to make a cell phone work on a cell phone. I wanted to expand the story more than I did, but I was afraid of ruining my progress with the code and switching of scenes that I had done. Going through the different screens that you see in the process of opening your phone, I wanted to explain the globalization of cell phones and how it could be that society now relies on them heavily. Overall I am happy with how this turned out, and am glad that I was able to export it as an app.

Video of app working: https://drive.google.com/file/d/1xfqJBbrvvQmrUWSGH3TdwRY2h8yC4wn-/view?usp=sharing
Zip file of Unity Project: https://drive.google.com/file/d/18iTmbZQgRhPqMF2Xr5gkfHr6cEX8roVO/view?usp=sharing

**Comment by yylinj** [comment]
I'm continuing with my exploration of Nutella as an object and a symbol. I edited the texts a bit to make them more coherent, and visually clearer. The process of making it into an APP is a bit harder than I thought it would be. But the most I'm struggling with was placing the scene under the AR camera and making them displayed in a normal way. I moved the camera first, so then everything got twisted and not functioning. My objects are also way too big, so I have to put them all into a parent canvas and scale them down a lot. 
I also experienced minute bugs and errors like functioning buttons or being unable to actually download the APP on my phone. They are all results of small mistakes like forgetting to click a checkbox, forgetting to ask permission for the camera, or forgetting to link the camera with the canvas that contained the button. But the debugging process was quite excruciating. But after fixing them, I think I'm happy with the app. I just saw the comments for my last assignment. I want to make the pictures floating in the background also match the playful color theme. My idea is to add maybe a colorful TV and have the pictures appear on the TV, but I'm afraid of more bugs and the time is a bit constraining. So I just edited the texts. 

Here's the video of the app working: https://drive.google.com/file/d/1r-H-zhS9a1USRcJe3MBEHatdBLZ6bqNK/view?usp=sharing
Zip file of Unity Project: https://drive.google.com/file/d/1-we-F4kMH5xW2hTAB2XRQm9pMqc1U3UI/view?usp=sharing

**Comment by miafan716** [comment]
Compared to the first assignment, I added more media for interaction. First of all, I make the font size and text box larger so that the text could still be clear on a mobile phone screen. Then I edited a few interactions. The first part about when Kirby is made, I added an animation on Kirby so as to make it more close to the user. The second part about how it was made, I introduced more his friends such as bulbasaur and squirtle and explained why Kirby is the most unique one. The last part about its consumption, I added more images and make them slide from one side to another side. 
However, you can see from the latter half of the video that there are still a lot of bugs that I didn't manage to figure out. The first one is the position of the bulbusaur and squirtle is very different than what I saw in Game mode on computer. Also, the Kirby game series can't have the same 3d effect and proper size in a screen space instead of the world space that I used in Assignment 1. What's more, I didn't figure out how to call two animations on one game object using different buttons. I tried the button onclick function to activate the animation clip and add animation arrays on the animator itself. But both didn't work. 

Here is the link to the video demo: 

https://user-images.githubusercontent.com/55967980/157334183-562dad41-0605-491b-9109-d03899129863.MOV


Here is the link to the Unity zip file: https://drive.google.com/file/d/1G9FQ1eQFFZPUUt17IxI8rxOkoWSbCz2A/view?usp=sharing

**Comment by Lkochar19** [comment]
Link to Unity zip file: https://drive.google.com/file/d/1zd04HKc1SMyYrZx2cF25NK8dDFTlVu0V/view?usp=sharing

**Comment by fractalessense** [comment]
The video I uploaded is a screen recording; through the experience is AR, it doesn't matter where one is. That is the fun: it puts you into this artificial room no matter where you actually are. Better, though, to be in a bigger open space. I guess, also, this project would work well as VR, though I designed it as AR and I think it works great. Alternatively, this virtual space could be made to represent and mapped on to an actual space. Like if I ran this AR experience in the yoga room itself and it was perfectly mapped.

I created a prototype 3D space in which one can explore objects and read text (as if projected onto a wall). The space is a 3D scan that I made of the yoga room. In the texts, I explore the possibilities of this 3D scanning and interacting technology for collaborative use and the new-age remix of cultures.

file:  https://drive.google.com/file/d/1hqF_YCdX4koV5VJsMd5ok6UBR5u5nqvt/view?usp=sharing

video:  https://drive.google.com/file/d/1ZXxum8wPRREBwBK38PE4jPj80wF_HR5O/view?usp=sharing

**Comment by joshblecherman** [comment]
[Link to video](https://drive.google.com/file/d/1JnDBnVqZnr9IX9NxjtwWDbHZcC8cusWH/view)

I created a 3D space that displays one of the Lewis chessmen as the 3d model. Behind it is 4 images that put it into context with consumption (the current World Chess Champion), production (a walrus, because the chess piece was made from walrus tooth), and communication (where the object was found, the current style of the chess king). In terms of communication, where an object was found tells a story about its origins. Also, the style of the design tells a story to the player regarding the type of piece it is, and it reflects how people believe a king should look: tall, royal, and adorned with a cross.

**Comment by aidanmassie** [comment]
Link to Video --> https://youtu.be/dHCzrLWu8tU
Unity Zip --> https://drive.google.com/file/d/1HfEdjT8qBO0ZD_kzkK7AhzlJDTri6f9G/view?usp=sharing

On this project, I struggled mightily to get Xcode working from the get-go, but was finally able to get something working. I took some feedback from the previous exercise and made changes to it, like making the viewing of the building more fluid. I adjusted the text and images around so the viewer could look at them while looking at the building, rather than having to search all the way off to one side or another to look at the images or read the text. I moved the buttons around as well. I also added an animation to the building to give it some more life, with a little spin and jump that allows users to see all the way around it. Unfortunately, I wasn't able to get the music working in the app itself--it worked fine on the Unity editor, and every forum I found online had no conclusive answer for my version of iOS/my iPhone that solved the problem. Overall, I'm very happy that I was able to get it working in the app (finally!!!!) and I feel good about its appearance as well.

**Comment by vb1424** [comment]
https://drive.google.com/drive/folders/1rJUAqt6FeuRzGGcol04DJE-aTUKMqew-?usp=sharing
^Link to a Google Drive file with both my zip file and a screen recording of my app!

**Community iOS App- AR Experience!**

This project was a pain to work with. I had so many struggles leading up to this satisfying moment of its completion. My struggles came from difficulties with my camera. My scene was set up nicely, my code all well written... but something was going on where every time I transitioned from Xcode onto the app on my iPhone, the camera was grey and then once I fixed the grey, the camera was showing the scene but it was frozen within it. By this I mean, it looked like there was just a screenshot of the scene on my phone but I could not move around within it. After weeks of attempts, and from particular assistance, it was decided that I should download a new update of an XR Plugin, and keep attempting at moving my camera and AR Origin around. At last, at sweet last, my app is perfectly functioning! My sound works, the buttons all work, and it came out looking (I'd say) pretty cool. My intention with this project, naturally was to build off my project from the previous Augmented Exercise (the relationship between community and kitchen). I took the feedback I was given, and used it to make some particular changes to my scene, and thus, the message it conveys to the audience. The message is more well-defined now: I am placing more of an emphasis now on what the meaning of community is, and then I use the act of cooking and eating as a universal way that community is formed. Thus: community is 'kitchen'. This is not to place extreme significance on the physical essence of a kitchen itself (as given to me in feedback, this emphasis would lead to questions regarding the role of gender in the kitchen and in comparative cultures). Kitchen is used as it has a universal understanding of being the place where cooking is done and eating may be consumed. It has more of a representational purpose... to place physical structure to the idea of a place where community is formed through the act of cooking and eating (of course). To find out more about why community is so related to the act of cooking and eating... check out my project! (Haha ;) ). But I'll give a hint: it is through the notion of individuals uniting and shedding their separation. Then the question arises, how is this done? And thus... well, I guess you will have to find out for yourself! Check it out and enjoy (make sure to put the sound on... it is kitchen noises- this is also something that I changed from my original project- to note). Before finishing this paragraph, it is essential to note that the significance of the 3D models and Sprites that I have included in my app/scene can be found in my description for my Augmented Exercise 1. But, in case you do not feel like traversing all the way to that link, I will put it briefly here for your ease: The images on the walls (my Sprites) are images of different civilizations sharing in feasts and meals (showing that this form of community has been around for quite so long). The little bronze statue has been taken from the Louvre museum, it is of Sumerians sharing a meal. Sumerians were from the civilization of Sumer: one of the first civilizations in the world. The following 3D model is, of course, of a modern family sharing a meal. Finally, the kitchen is an Italian style kitchen, and it is so just to draw a unifying string around the idea of this universal form of community that is found across all cultures.

Danke :)

**Comment by aashnasoni** [comment]
I definitely struggled with this assignment as navigating Unity was quite difficult with what I wanted to achieve through this exercise. I wanted to allow the user to have first person access into the museum which would move around as the device moved, however, after many attempts to achieve that, I was still unsuccessful. This project  is largely based on understanding the Koh-i-Noor diamond and its origin, preservation and consumption which is shared through texts and pictures. I think it’s interesting to see how the 3D museum structure can be viewed through a mobile screen, allowing the user to visualize it in different contexts. I do feel that I would’ve wanted to scale it up further and play with the interaction a little more, however I really struggled with using Unity in general and getting that part of it to work. I am happy with the way my project from Augmented Exercise 1 was able to be interacted with through an AR camera, allowing a tap feature of the button to change the images. Placing the scene in the camera was difficult on its own, but connecting this through a script that would allow buttons to move between different scenes was out of my scope, even after watching multiple tutorials online. I would want to develop this further after being a little more comfortable with Unity because I feel that bringing this gallery experience to a virtual setting would be cool. 

Link to drive: https://drive.google.com/drive/folders/1ivmiNvSzL_yFbLV2P1WCuvUs0gbGuZBG?usp=sharing
(I'm also unable to run the code on my phone because the version of iOS isn't supported with Xcode for some reason)

**Comment by chloeumlaut** [comment]
As you know, this project was a long journey for me with lots of bugs and starting over. My goal was to convert the user's environment into a wind farm with informational panels that would tell them information about wind energy as a renewable energy option. Though it took me a long time to figure out how, I did this by making a series of 3d object wind turbines, and a canvas with my "informational panels" my prefabs that appear when a plane is detected. After I figured out how to do this, I of course dealt with tons of sizing and spacing issues, but eventually got it to a point where I think everything fits well on the screen. There are still some bugs that are bothering me, the plane scanner doesn't seem to disappear after the plane is detected which obscures the view of the informational panels a bit, and if I were to continue with this I'd add more of a "start" screen or button guiding the user to the text, but overall I believe the project translates. 

Link to video and zip file in this folder: 
https://drive.google.com/file/d/17vEnqz4oHfXGj41_9JCyQPkh-jHC9MIn/view?usp=sharing

**Comment by Mg5970** [comment]
This project was a tumultuous journey of going through multiple iterations and trial and error troubleshooting. Files were lost multiplied restored etc. Xcode errors are much more technical than unity errors but this exercise definitely taught my how to search for solutions on the internet for both unity and Xcode which I know is a very importing skilling coding in general so it was great to learn it. The transition to an AR app meant a lot of resizing things for me and finding errors that were not an issue until the scene became intended for AR use. One button was removed for UX sake and I also learned how much more user friendly everything much be once it becomes a mobile application. All in all I was just very glad to finally see my work realized on the app it was a very gratifying end to a long journey. 

 Link to video and zip file in this folder:
https://drive.google.com/drive/folders/1yrZZbTrnElva84q0ARUf8Bgd87utzoYM?usp=sharing

**Comment by palomabrites** [comment]
I definitely struggled with this assignment and getting the code and the ar function to work. At first it opened as an app on my computer and once I switched it to my phone the camera part stopped working however I was able to get the button to work and cycle through the images. I was interesting to transition the project to ar because of the resizing needed and a lot of back and forth between unity and my phone screen to see if the objects fit correctly. I also had some issues with xcode that wouldn't allow me to run the app on my phone originally but I was able to correct it. 
https://drive.google.com/drive/folders/1PuH7a7og99yQmzQCwI1-rCRGyiZxn7VU?usp=sharing

**Comment by Dav1dyang** [comment]
<img width="1840" alt="Screen Shot 2022-10-25 at 8 33 56 PM" src="https://user-images.githubusercontent.com/55843005/197854027-cfcdef04-442b-41c0-a291-30e319b0dffb.png">

The entire process of getting Unity AR to work on my iPhone 12 is excruciating. After spending a lot of time during the last class trying to figure it out, I decided to take a break and not think about this. After the break, I took Pierre's advice and installed AR foundation 5.0.2 and ARKit Plugin 5.0.2. The project worked. I hope I can be more fluent in coding and Xcode development so I don't have to go through Unity. Despite Unity's convenience, I feel like I have less control over the software and less access to the hardware.

Here is a video link to the working project: https://youtu.be/IjPy337wk78
And here is the link to the Unity project file:  https://drive.google.com/drive/folders/1RPdIrXqrHm1xCcIR7Ypq_VgjKHAunbIB?usp=sharing

**Comment by kenparks12** [comment]
Like everyone else, I had quite a bit of trouble getting everything to work. After many tries and updates, I was able to load my scene into unity, however, I ran into an issue with using all my previous UI components, so I had to delete/reorganize quite a few. In the end, I was able to create a simpler iteration of my first project so I am happy with it. 

[Video Link](https://youtube.com/shorts/UitaatQHZ5E)
[Drive Link](https://drive.google.com/file/d/1JAUt8mZ2INfADseo_ZEiD8GTJVS-WVm1/view?usp=sharing)

**Comment by IanPort** [comment]
I knew I would face difficulty working between multiple computers to access Xcode, but I managed most of the errors to get my application running. Unfortunately, the farthest wall texts from the camera aren't clearly visible and with some more time, I hope to sort the issue. I realized through the initial errors, where asset, component, and object files were missing or random, that my file organization needed to improve. But I felt accomplished that the animation (coding component) remained in the phone application. I'm excited to create more content using Unity...!

[Exercise Zip File](https://drive.google.com/file/d/1dlvmfagOD6426wBBGmj8qs3bVVIxTCTe/view?usp=sharing)
[Video](https://drive.google.com/file/d/1MKnzib1DphlGK0x1-2tCQaumtz_eiU3T/view?usp=sharing)

**Comment by izzyrieken** [comment]
This version is still not completely fixed. I am struggling to make the AR camera move even after updating my AR Packages to 5.02 (Thank you David). Luckily Xcode works pretty smoothly and didn't have to many problems there. 

Video: https://drive.google.com/file/d/1ghi3HBkTi9VlYLgylDmQAUEjgNzIOLKi/view?usp=sharing
Build Folder Zip: https://drive.google.com/file/d/1ci9MXFFCdTeoSLTc552cQI263dGSa35N/view?usp=sharing

**Comment by luyumenq** [comment]
It feels exciting to see my virtual gallery coming into reality! Basically there seems to be no technical difficulties (version mismatches, etc.) while shifting from my original version in both Unity and Xcode. I have discovered something very interesting, that my Canvas in Screen Space also appeared as horizontal instead of a vertical screen space to locate my button.

https://user-images.githubusercontent.com/72938590/197983608-9f553636-83a0-43a5-b96b-6032f07d4350.mov
Folder URL: https://drive.google.com/file/d/1DGqwYNqHhmszcl8qLOScc2ubF7Pe3Fp0/view?usp=sharing

**Comment by kylewenchang** [comment]
I converted my last project about bottles into AR by copying the assets/prefabs to a new project where I placed them in the AR camera's worldspace. Rather than having click/tap interactions, which I thought might be clunky in AR, I instead put all the "slides" laid out in space with the information always presented rather than being behind a click. Like many others, I ran into issues with Unity and Xcode, having to start a new project after hours of updating. However, once compilation issues were resolved, it worked! Even though it didn't change much from the last project, it's rewarding being able to see it in AR through the camera.

[Video Demo Here](https://www.youtube.com/watch?v=uVyYNUyxbyA)
[Project Folder Here](https://drive.google.com/file/d/158gkdjgSURFCGeDud3Iio98QMnkSzQiD/view?usp=sharing)

**Comment by olivestew222** [comment]
ogs220- Olive Stewart
link:​​https://drive.google.com/drive/folders/1ICNVsGauhDhIFP36vyAzE6BX5R39WySo?usp=sharing

I recreated my museum space to act as an AR experience. It gives the user the illusion of walking through a gallery space, which is quite dark and moody to compliment the object (The Weeping Angel). I followed my classmates advice and renamed the buttons so they made more sense. I left one button floating onscreen so that the user may have constant access to the main information. However, I put it out of the way as not to block usage. However, since the user can move through the scene on their own, I opted to remove the click number to move spaces function. Instead I moved the other buttons so users could search for them and click on them closer to relevant objects. Overall, this project was frustrating for me to complete because Xcode would not work on my computer and no amount of searching could remedy my problems. I have classes that overlap office hours so I had trouble finding time to get help. On top of this, my laptop broke so I needed to rework my issues again on the school computers. It was quite rewarding to see the final result when this project finally worked out. I hope to continue honing my skills

**Comment by emersonmack** [comment]
https://drive.google.com/drive/folders/1iscRgUa1sQU-GIV01mFzbJcet23Fc8yZ?usp=share_link
Here is a link to my drive folder with my submission. For this project, I revised my Hadrian Gallery from the last time. I simplified the design while also spreading it around the space. The original focused primarily on the back wall while this adaptation engages each side of the gallery. I struggled with the lighting design as it looked different in the AR version than in the unity version and I would like to perfect that next time. In reflection, I am very proud of my progress and understanding of the software as well as my growth from our previous assignment.

**Reply by periode** [reply]
@emersonmack 

You did a great job at porting your gallery from the desktop to mobile!.

As mentioned in class, there are a couple of things you could look into to improve in the next version:

- Think about whether you do need the walls or not: what purpose would they serve? How do they open up or delimitate space? In terms of presentation, maybe you also want to look into using different kinds of fonts and colors to provide a more immersive perception of your audience.
- Think about the amount of information you're displaying on Hadrian (I find it a bit too sparse), and _why_ you're displaying it: why should people care about Hadrian? Is it cool to build walls?
- Think of the means of interaction: how helpful or limiting are buttons? How well would buttons fare if you have more contents?

Specifically in terms of contents, I'd advise you to think about what other objects you can add to your collection for the next assignment: more statues? more walls? another dimension of your current object?

Grade: 92 (A-)

**Comment by aacarcrash** [comment]
<a href="https://drive.google.com/file/d/1hkkHyiQYERKy-X4AD9EdfbqCLKVxR1X3/view?usp=share_link"> Drive Link </a>
<a href="https://www.youtube.com/watch?v=o7Muj-um49M"> YouTube Link </a>

Instead of doing a chronological spatial exploration as in I had done for the initial assignment, I chose to layout the objects differently. This was mainly due to, as Pierre mentioned in class, the scale and spatiality work in different ways when objects are put into AR. Hence, I shrunk the models and placed them around the Nandi Bull so the audience could explore it conveniently As there is not as much spatial exploration as before, instead of changing the texts using box collisions I setup a raycaster to change the text upon clicking on the models of the locations themselves. I also made them rotate so that audience could be attracted to them and look at them from all angles while not making an effort. I made the bull appear stationary to make it stand out, let the audience walk around it at their own pace (as they would in a museum) and to remove visual clutter from the scene. 

On a technical aspect, I think I am running into issues with what the origin of the AR scene is. Sometimes, the AR origin also appears to shift mid-way through the app. I am not sure how scale and distance works either. What I simply wanted to do was create a layout that could fit on a dining table and it spawns in front of you. Instead, I was met with a lot of clutter and glitches with the origin. Technical aside, design wise visual clutter seems to be an issue as well. I don't know how to lay out objects in AR that doesn't involve crowding them in a small room. In an ideal scenario, they would be placed out well apart and the audience could explore it comfortably. Instead here, the objects in AR layered with the objects in our reality creates a very discomforting, crowded and overstimulating experience. I would like to understand how to design for AR better as well.

**Reply by periode** [reply]
@AakSin 

Your spatialization is very interesting, and you engage directly with what it means to inspect a digital object: the rotation works well when you scale down the objects. That being said, there are questions about interaction that are left unanswered, and which you could touch upon in your next assignment. How do you let visitors control that rotation, and other physical features of the object? How do you balance that with the need to read static content?

As you think about modes and ranges of interaction, think also of the defaults: how does it look like the first time? how is that supporting the majority of visitors who might not change those defaults?

In terms of content, I am curious in which directions you are going to expand your collection: is it about indian culture, history and present? about orientalism? about cultural spoliation across places and/or eras? etc.

Grade: 98 (A)

**Comment by Yupu-Chen** [comment]
https://user-images.githubusercontent.com/112383893/221973763-58b5a29a-3f72-448c-8c82-2d0c00716642.mov

Link to the zip file: https://drive.google.com/file/d/14AjMpgaB_A2y4jH70bdscuasmN0ipNS5/view?usp=sharing

For this AR project, I get rid of the garage setting as in the first project. The purpose of doing so is to create a sense of combination of the reality and the virtuality, which differentiates AR from VR. The objects and UI elements can be better incorporated into the environment of the user.

In terms of the design and the content. I changed the font and word background to fit the modern style of Boeing 787 better. I also changed the content in consumption as according to Pierre's feedback, the topic of carbon emission and global warming might be a more interesting one to talk about in the consumption part of the aircraft. Also, instead of using a First Person game approach as in the first project, I lay all UI elements out so that the user can better interact within a limited space.

Technology-wise, I wanted to anchor the object and UI elements to a plane (i.e. a table or a wall) in the reality, but this caused the malfunction of buttons. I also tried to allow the user to place the object to any plane in their environment. It worked, but interfered again with buttons. Therefore, I deleted these features and hope to realize them in the next project.

**Reply by periode** [reply]
@Yupu-Chen 

Your translation from a desktop gallery to a mobile AR one is very well done! The panel in front of the audience is very clear, and provides a subtle progress from production to mediation, with a very varied range of infromation and support media.

As mentioned in class, I have a couple of feedback points about the layout:

- Taking full advantage of AR would imply having multiple planes, intersecting each other, and building up a real 3D space, rather than a 2D space
- You could use more whitespace: the different blocks of the exhbition could be more differentiated by spacing them out a little.

I'm looking forward to the next step of your collection: will you keep on focusing on the concept of flying? on transporation technologies? on travel and globalization? or something else?

Grade: 98 (A)

**Comment by Opal42** [comment]
Apologies, there's no video because I can't emulate the app on my laptop directly, and I don't know how to record it off my phone directly. 
Source files here: https://drive.google.com/drive/folders/1KFIA3FDho_weSk3h1emvZfaUYGfX4Y3s?usp=sharing

This is a pretty straight-forward port of my first app onto mobile, the original UI was already designed with this type of interfacing in mind, so little has changed there (besides an aspect ratio shift). The UI is designed for a hypothetical world in which the object per say was an actual physical object, rather than a digital one--augmented rather than virtual reality. I've also added in a rotation animation for the object to better highlight the difference between being "enlightened" and "ignorant". 

Unfortunately, as of right now I continue to have extreme issues with drifting, which I suspect is reflective of a larger issue with either my phone's hardware or ARCore's compatibility issues with Android 12. When running a visualiser through the PlaneManager, it reveals not only a wildly inaccurate series of boundaries for planes, but also generates many intersecting planes at strange angles where there ought to be none. I've tried attaching an ARAnchor to the Fishbowl, as well as instantiating it at runtime attached to a plane (although I'm not confident that was scripted correctly), but neither have given better results.

**Reply by periode** [reply]
@Opal42 

You did very nice adjustments to your gallery, keeping it straight to the point. Since your object is so minimal, my main feedback would be to consider removing the background, and using an AR Raycast manager to allow the user to place the object themselves on a detected plane.

This would allow the visitor to also move around (ignoring hardware/software drift for now) and really feel like they are investing an unknown artifact, rather than staring at it from a static perspective.

Looking forward to seeing how you expand this collection: the theme of scientific knowledge discovery and historical artifacts is a very rich one!

Grade: 98 (A)

**Comment by hannah-y-zhao** [comment]
[Link to video and files](https://drive.google.com/drive/folders/1R7XRFHXqQO5J0DB5A1usWTMbHIA4xnZm?usp=share_link)

I took most of the same elements from the last assignment, but I tried to expand on the "Future" part of it for more in depth questions. I originally wanted an input field where users could input their own thoughts on a potential future, or have users interact with different text through dragging but I didn't know how to do that. In general, I hope to add more user functions to make a more personalized experience; I also want to be able to take better advantage of the AR medium, and lay out my elements in a more "natural" way.

I also found that testing even a little change through AR took a very long time. Otherwise, it was really cool to see it appear on phone.

**Reply by periode** [reply]
@hannah-y-zhao 

You did a great job at moving from a 2D layout and into more of a 3D layout, and the fact that the viewer finds themselves surrounded on all sides provides a fun element of surprise. In terms of layout, however, I'm not certain about the text being toggled on/off while the underlying white solids remain there. One of the unique aspects of digital media is this ability to modularize the display of content, and so it could be interesting to think of the storytelling of your app along these lines. For instance, the visitor can be presented with just a coin, then upon interacting with the coin they get to see the three round pieces with origin/culture/future, etc.

In terms of user input, you can use a built-in UI element called Text Input, to which you can attach a script and store the text being input in a variable.

Looking forward to seeing in which direction you develop this collection.

Grade: 98 (A)

**Comment by mariamabdelrazekk** [comment]
video: https://youtube.com/watch?v=-vpviRaESQk&feature=shares

Unity file: https://drive.google.com/drive/u/0/folders/1xR6Ecs5Acm0BKHZA2Ii0PopnadfNq1Dl

For my second Augmented Assignment, I basically used the setup I had from last time and tried to picture it in a physical space. I faced a some difficulties during this assignment, mostly related to scale, the project initially was too big to be viewed in real life, therefore, I had to scale it down by half. Moreover, while moving through my space, as you can see in the video, my unity project physically moves with me in real life, which makes it harder to view in a small space. I thought that the buttons working on the application would have been my biggest challenge, however, thankfully, they worked.perfectly fine on first try,

**Reply by periode** [reply]
@mariamabdelrazekk 

Your layout of the Rosetta Stone is very impressive, if only due to the relative size of the stone compared to the walls of your gallery. On one side, the way these walls are set up help create a linear storytelling, and deliver information in a straightforward way. On the other side, their size and scale makes the whole exhbition a bit narrow, while assuming that the viewer will be able to walk 2+ meters to reach the stone (which might not happen in all environments).

That being said, I think you did a great job at providing all around information about the stone, and I'm curious to see in which direction you are going to bring your collection: are there any other artefacts similar to the Rosetta stone? what other artefacts do we have that relate to language and translation?

Grade: 98 (A)

**Comment by daniaezz** [comment]
[link for video and zip file](https://drive.google.com/file/d/1JkKcnEXdJBhZrKbaZ4L4inLouE56pDyv/view?usp=sharing)

For this AR project, I tried to make it so that the user can walk along the space where the object is located, but when I tried it in my room it proved to be difficult because of the limited space, but it worked better when the space available was larger. However, even with the larger space, I found that the objects move back and forth with the camera a little, which makes it a little difficult to walk behind the walls and go around the sculpture. 

To display the information, I decided to put it on plaques in such a way that the user can walk around and read more about the sculpture and the space it exists in. I also positioned the plaques in such a way that they would lie in the “rooms” so that the user could get to walk around the barriers when they triggered them.

Overall, I feel like the optimal implementation of this AR app would be in the actual space where the sculpture exists, because then the user can get to explore the space and experience how it is in reality and then get additional information about it using the AR app.

**Reply by periode** [reply]
@daniaezz 

I found your spatialization of your gallery very impressive! The appearance of the fences around the space is an emotional moment that allows to connect to the story on a different level. From the start, the texture and geometry of the text displays is also very diegetic and really bring an impression of coherence and consistency.

That being said, there are a few aspects that could have been polished further:

- I've found that the relative position of the objects to the camera was a little bit too high. If your intent was to put the viewer in the shoes of a child by showing how it would be to stare from their height, I would have rather looked into making the objects bigger, rather than putting them higher.
- While the brown slabs are great, the floating pictures stand in very stark contrast, and break slightly the immersive feeling you manage to create. Perhaps you could have put them on smaller versions of the brwon stone?

All in all, great job, and I'm curious to see how you will develop this into a collection.

Grade: 99 (A)

**Comment by cz2109** [comment]
Drive link for video and zip file: https://drive.google.com/drive/folders/1Lb6njkBERgu6JVRzIBJWS3uXF7-7bIgG?usp=sharing

For Augmented Exercise 2, I kept the layout of the scene almost identical to the first exercise. Scaling the gallery in AR was pretty straightforward and did not yield much difficulty, only because I had kept in mind the abstract metric guidelines (where 1 = 1m) from earlier on in the class. The AR elements that did pose some challenges were increasing the field of view since there were glitches in the walls and positioning the AR camera to best fit that of a first-person viewer.

Beyond AR, I wanted to edit informational inconsistencies from my first project. I streamlined the logic between the sculpture and potential queer interpretations with a consistent theme of anonymity in contrast with my first exercise which did not have any conceptual transitions.

This exercise was much easier to grasp than the first. I feel very comfortable manipulating a scene and displaying such in an AR app; however, I still lack a lot of confidence with scripting for a scene. Exploring AR made my efforts from exercise 1 feel more tangible, and I appreciate that I had an opportunity to not only discover, but hone my knowledge in AR, which was always something I believed to be an incredibly daunting skill/concept.

**Reply by periode** [reply]
@cz2109 

The translation from desktop to mobile is straightforward, but nonetheless works very well. In particular, the new panel on queer art works well insofar as it connects the age-old with the hyper contemporary and could raise interesting questions, but you should have made this connection more obvious, beyond a simple juxtaposition.

In terms of layout, there is a thin line between the intimacy of a small space and the claustrophobia of a tiny room. Perhaps you could look into ways to soften the edges and corners of the room, or think about ways to open up the room, perhaps by having alcoves floating independently. Unity has a package called ProBuilder which is great for 3D modelling.

I am looking forward to seeing how you further your collection, and draw more parallels on artistic representation of intimacy in various societies, and beyond.

Grade: 94 (A-)

**Comment by chaerin-d** [comment]
Link to project: [http://localhost:3001/](url) 

For this project, I added a home button, which gives access to people by just clicking the header "SSAM" to go to the main page, I got a comment about having clear division for the illustrations, so I made an individual box for each illustration. One of the most entertaining parts that several people commented about was the mini-game(screenshot), I thought if I added music while people were playing, it would be more entertaining, so I decided to use music that I created and added from SoundCloud. I tried to practice using "react" so I used insert {} and replaced the image which is a benefit that I can make a guideline, which wasn't available in HTML. 
I hope I can have more interaction with the user like vegetables pop up as AR effect through their phone as I continue to learn about UI/UX Design. 

*Screen recording and all the files are in Google Drive. 
https://drive.google.com/drive/folders/1wkV8H5kQIqe0fafJXKvmiThJRet2vYaL?usp=sharing

**Reply by chaerin-d** [reply]
http://localhost:3001/
